book/02-git-basics/sections/recording-changes.asc

```diff
@@ -489,18 +489,18 @@ $ git commit -a -m 'added new benchmarks'
  1 file changed, 5 insertions(+), 0 deletions(-)
 ----
 
-Notice how you don't have to run `git add` on the `CONTRIBUTING.md` file in this case before you commit.
-That's because the `-a` flag includes all changed files.
-This is convenient, but be careful; sometimes this flag will cause you to include unwanted changes.
+Pansinin kung paano hindi ka kailangan na magpatakbo ng `git add` sa `CONTRIBUTING.md` na file sa kasong ito bago ikaw mag-commit.
+Iyan ay dahil ang `-a` na flag ay nagsasama ng lahat na nabago na mga file.
+Ito ay maginhawa, ngunit magingat; kung minsan ang flag na ito ay magdudulot sa iyo ng pagsasama ng hindi inaasahang mga pagbabago.
 
 [[_removing_files]]
-==== Removing Files
+==== Pag-alis ng mga File
 
 (((files, removing)))
-To remove a file from Git, you have to remove it from your tracked files (more accurately, remove it from your staging area) and then commit.
-The `git rm` command does that, and also removes the file from your working directory so you don't see it as an untracked file the next time around.
+Para mag-alis ng isang file mula sa Git, kailangan mong alisin ito mula sa iyong nabubaybayan na mga file (mas sakto, tanggalin ito mula sa iyong staging na lawak) at pagkatapos i-commit.
```

Comment:
* ``nabubaybayan`` to ``nasubaybayan``
* you can also change ``sakto`` to ``tama``

Reply:
done change